engine/platform/dos/vid_dos.c

```diff
@@ -543,4 +543,9 @@ qboolean SW_CreateBuffer( int width, int height, uint *stride, uint *bpp, uint *
 	return true;
 }
 
+qboolean R_GetWindowHandle( void **handle, int type )
+{
+	return FALSE;
```

Comment:
Oh, you can use just `false` in engine and renderer code. Not sure where `FALSE` could've came from.

Reply:
Grepping engine source code for `FALSE` and `TRUE` sounds like yet another Win32 API retardness.

We don't do that here.